openpype/hosts/maya/plugins/publish/validate_rendersettings.py

```diff
@@ -211,14 +213,20 @@ def get_invalid(cls, instance):
                     cls.log.error("Wrong image prefix [ {} ] - "
                                   "You can't use '<renderpass>' token "
                                   "with merge AOVs turned on".format(prefix))
+                default_prefix = re.sub(
+                    cls.R_AOV_TOKEN, "", default_prefix)
+                # remove aov token from prefix to pass validation
+                # first resolve it and then remove if dangling
+                default_prefix = default_prefix.replace(
+                    "{aov_separator}", instance.data.get("aovSeparator", "_"))
+                default_prefix = default_prefix.rstrip(
+                    instance.data.get("aovSeparator", "_"))
```

Comment:
Why first resolve it and then strip the right hand side aov separator tokens, that seems redundant and maybe even not as explicit as one might want. If it's stripping it anyway shouldn't it just replace the explicit `{aov_separator}` placeholder with an empty string?

```suggestion
                # remove aov token separator
                default_prefix = default_prefix.replace("{aov_separator}", "")
```

Additionally, since the image prefix is hardcoded in the validator anyway (doesn't seem to allow to be overridden by settings) this "fix" could even be made more explicit by explicitly replacing the end of the default image prefix alltogether?

We could even get by with just a single `re.sub(re.escape("{aov_separator}<RenderPass>") + "$", "", default_prefix)`? Or even as simple as `prefix = prefix[:-len("("{aov_separator}<RenderPass>")]` to just strip off the hardcoded end or `prefix = prefix.split("{aov_separator}")[0]`?

Reply:
@antirotor just pinging you here in case you didn't notice